### ISSUE TITLE
[Bug]: Footnote Numbers in embeds not displayed properly when using the "clean-embeds" cssclass in reading-view

### ISSUE THREAD

**Issue by Gewerd-Strauss** [chrisgrieser/shimmering-focus#196]
### Bug Description

When embedding a note containing footnotes, these respective footnotes are prinited at the bottom end of the embed, before the delimiter bar. 
In live-preview and source modes (where the cssclass does not take effect), the footnote numbers within an embedded note are displayed as proper numbers regardless of whether or not the cssclass is active or not, as it does not take effect in any mode besides reading-mode. 
When using the reading mode with `clean-embeds` active, the numbers of the footnotes at the bottom of the embedded note's space gets reduced to some very small dots, cf. Image 2. 

When not using the `clean-embeds`-css and switching to reading mode, the problem does not occur, cf. Image 1.

I would be very happy if this can be resolved, and I am happy to give more details if required.

Thank you,  
Sincerely,  
~Gw

### Relevant Screenshot

![grafik](https://user-images.githubusercontent.com/80548781/180874009-0171acb3-0978-40eb-87e7-e77e6ede34f9.png)
Image 1: Display of the footnotes within an embedded note, reading mode, no cssclass active

![grafik](https://user-images.githubusercontent.com/80548781/180874192-9bb27b49-6b56-4b5a-a8a8-6bee10b19fef.png)
Image 2: Display of the footnotes within an embedded note, reading mode, with `clean-embeds`-cssclass active

### To Reproduce

1. Create file1 with several footnotes
2. embed file1 into file2
3. switch to reading mode within file2
4. add the clean-embeds-css and repeat step 3.
5. compare

### Obsidian Version

0.15.8

### Installer Version

0.14.15

### Font Size

22

### Which editor are you using?

- [X] New Editor
- [ ] Legacy Editor

### In which mode(s) does the problem occur? (if applicable)

- [X] Reading (Preview)
- [ ] Source (Edit)
- [ ] Live Preview

### Which operating system?

- [X] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### Plugin-Specific Problems

- [ ] I took note of the [specific instructions regarding those plugins](https://chrisgrieser.github.io/shimmering-focus/plugin-specific-features/#instructions-for-specific-plugins).

### Checklist

- [X] I updated to the latest version of the theme (Settings → Appearance → Themes - Manage → Update).
- [X] I made sure that the issue is related to this theme by checking that the problem does not occur with the default theme.
- [X] In case I am missing an UI element, I have checked that it cannot be re-enabled with the [Style Settings plugin](https://github.com/mgmeyers/obsidian-style-settings).
- [X] If the bug occured after updating the theme, I read the last couple of entries in the [changelog](https://github.com/chrisgrieser/shimmering-focus/blob/main/docs/Changelog.md#changelog) to make sure the bug is not an intentional change. (In that case, there will most likely be a style setting to turn it off again.)

**Comment by Gewerd-Strauss** [comment]
Additionally, is it possible to have certain cssclasses (especially `clean-embeds`) to also render in edit mode? I usually fully leverage the advantages of live-preview, but the power of clean-embeds is the only thing that usually forces me to switch to reading mode instead to correctly render. 

I am aware that right now according to the theme docs cssclasses only render in reading mode, but I'd like to know if it is _possible_ and could be implemented as an option perhaps.

**Comment by chrisgrieser** [comment]
Hi! As you may have heard, the upcoming Obsidian update 0.16 has a massive amount of changes for theme development, necessitating a complete rewrite of *Shimmering Focus*. This means:
1. A lot of bugs and feature requests will be obsolete. with Obsidian 0.16 and the rewritten *Shimmering Focus*.
2. As the theme has over 15,000 lines of CSS, the rewriting will take a considerable amount of time. Recreating the core features alone will likely take me a while, so I simply will not have much time to implement many feature requests, since the recreation of the past core features has priority.

Basically, this is the reason why I am – for now – closing this issue. If your bug still exists / your feature request is still relevant in the upcoming version 3.0 of *Shimmering Focus*, please comment here and I will re-open it. 🙂

**Comment by Gewerd-Strauss** [comment]
Alright that makes sense, no worries 🖖 
Good luck translating the theme, I am seriously considering staying on 0.15 for the time being until most my plugins have been updated - I have _a lot_, and they are all rather important to my workflows.
